### PR TITLE
Play restart stop

### PR DESCRIPTION
**Pull request by eprincev-egor**
Hello! in every project i have commands for build project or run server. And every time me need call it commands. For fix it trouble i offer it commit.

config  .platformio-ide-terminal file example:
```
{
    "icons": [
        {
            "name": "Test",
            "color": "Orange",
            "command": "npm i",
            "templates": [
                {
                    "name": "npm i",
                    "command": "npm i"
                },
                {
                    "name": "npm i --dev",
                    "command": "npm i --dev"
                }
            ]
        }
    ]
}
```

### CODE REVIEW

**Comment by eprincev-egor** [comment]
image

File: lib/status-bar.coffee [hunk 1]
```diff
@@ -81,7 +82,7 @@ class StatusBar extends View
             nextTerminal.toggle() if prevTerminal?.panel.isVisible()
 
     @registerContextMenu()
-
+
```
Comment (the-j0k3r) [inline lib/status-bar.coffee:85]: mmmm?! :confused:

Comment (eprincev-egor) [reply lib/status-bar.coffee:85]: what's wrong here?

Comment (the-j0k3r) [reply lib/status-bar.coffee:85]: what are the spaces for?

**Comment by the-j0k3r** [comment]
@ivankravets review?

File: lib/status-bar.coffee (hunk 1 above)
Comment (the-j0k3r) [inline lib/status-bar.coffee:85]:
```suggestion
```

**Comment by ivankravets** [comment]
There is "Custom Texts" setting. 
Screen Shot 2019-04-01 at 14 11 17

**Pull request closed without merging**